src/reader.rs

```diff
@@ -67,6 +70,21 @@ impl AsyncFileReader for Box<dyn AsyncFileReader + '_> {
     }
 }
 
+impl AsyncFileReader for std::fs::File {
+    fn get_bytes(&self, range: Range<u64>) -> BoxFuture<'_, AsyncTiffResult<Bytes>> {
+        async move {
+            let mut file = self.try_clone()?;
+            file.seek(std::io::SeekFrom::Start(range.start))?;
+            let len = (range.end - range.start) as usize;
+            let mut buf = vec![0u8; len];
+            file.read_exact(&mut buf)?;
+            let res = Bytes::copy_from_slice(&buf);
+            Ok(res)
+        }
+        .boxed()
+    }
+}
```

Comment:
We should not add this because this will perform blocking reads on the main thread. 

Instead we should probably restore the tokio integration 
https://github.com/developmentseed/async-tiff/blob/6a7a2dc038f7b84e8ada5ef16e2cb8c1e5504334/src/reader.rs#L70-L91 which will allow reading from a [tokio async file](https://docs.rs/tokio/latest/tokio/fs/struct.File.html), which I think performs file IO from a thread pool

Reply:
The problem I had was that the feature testing also wants to test with all features disabled, and I didn't exactly know how to deal with that... added a fallback in util.rs

Reply:
Also tokio's asyncread require `&mut self`, so I changed the impl to `tokio::fs::File`, where we can `try_clone()` on `&self`